modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItTablePersistenceTest.java

```diff
@@ -197,7 +197,7 @@ public void afterSnapshot(RaftGroupService service) throws Exception {
     @Override
     public BooleanSupplier snapshotCheckClosure(JraftServerImpl restarted, boolean interactedAfterSnapshot) {
         MvPartitionStorage storage = getListener(restarted, raftGroupId()).getStorage();
-        Map<ByteBuffer, RowId> primaryIndex = getListener(restarted, raftGroupId()).getPk();
+        Map<ByteBuffer, RowId> primaryIndex = null; // getListener(restarted, raftGroupId()).getPk();
```

Comment:
Could you please either fix it or create separate ticket if it's not possible because given ticket is disabled.

Reply:
I believe this could be fixed within the ticket under which this test is disabled

Reply:
Well, let's try.